src/ontodia/data/sparql/sparqlDataProvider.ts

```diff
@@ -22,6 +22,13 @@ import { SparqlDataProviderSettings, OWLStatsSettings } from './sparqlDataProvid
 
 export enum SparqlQueryMethod { GET = 1, POST }
 
+export type QueryFunction = (params: {
+    url: string,
+    body: string,
+    headers: any,
```

Comment:
Actually it's partial fetch parameters, I would prefer them to be finite and described.